overlord/snapstate/snapmgr.go

```diff
@@ -357,6 +357,7 @@ func Manager(st *state.State, runner *state.TaskRunner) (*SnapManager, error) {
 	runner.AddHandler("start-snap-services", m.startSnapServices, m.stopSnapServices)
 	runner.AddHandler("switch-snap-channel", m.doSwitchSnapChannel, nil)
 	runner.AddHandler("toggle-snap-flags", m.doToggleSnapFlags, nil)
+	runner.AddHandler("rerefresh", m.doReRefresh, nil)
```

Comment:
still not 100% sure about this name, my main issue is that the optionality is not very spelt out by this name

Reply:
after talking on IRC, we went with `"check-rerefresh"`